<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Angeltagebuch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Angeltagebuch.qrc">
    <normaloff>:/appIcon.png</normaloff>:/appIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAngelplatz">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <addaction name="actionBEenden"/>
   </widget>
   <widget class="QMenu" name="menuBearbeiten">
    <property name="title">
     <string>&amp;Bearbeiten</string>
    </property>
    <addaction name="actionNeu"/>
    <addaction name="actionNdern"/>
    <addaction name="actionLschen"/>
    <addaction name="separator"/>
    <addaction name="action_Suchen"/>
   </widget>
   <widget class="QMenu" name="menuDaten">
    <property name="title">
     <string>Da&amp;ten</string>
    </property>
    <addaction name="actionMarkierterAngelplatz"/>
    <addaction name="actionAlleAngelpltze"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>E&amp;xtras</string>
    </property>
    <widget class="QMenu" name="menuSprache">
     <property name="title">
      <string>&amp;Sprache</string>
     </property>
     <property name="icon">
      <iconset resource="Angeltagebuch.qrc">
       <normaloff>:/languageIcon.png</normaloff>:/languageIcon.png</iconset>
     </property>
     <addaction name="actionDeutsch"/>
     <addaction name="actionEnglisch"/>
     <addaction name="actionSerBisch"/>
    </widget>
    <widget class="QMenu" name="menuHintergrund">
     <property name="title">
      <string>&amp;Thema</string>
     </property>
     <property name="icon">
      <iconset resource="Angeltagebuch.qrc">
       <normaloff>:/hintergrund.png</normaloff>:/hintergrund.png</iconset>
     </property>
     <addaction name="actionWeiss"/>
     <addaction name="actionGelb"/>
     <addaction name="actionGraU"/>
     <addaction name="actionGRn"/>
     <addaction name="actionRot"/>
     <addaction name="actionBlau"/>
    </widget>
    <addaction name="menuSprache"/>
    <addaction name="menuHintergrund"/>
   </widget>
   <addaction name="menuAngelplatz"/>
   <addaction name="menuBearbeiten"/>
   <addaction name="menuDaten"/>
   <addaction name="menuExtras"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNeu"/>
   <addaction name="actionNdern"/>
   <addaction name="actionLschen"/>
   <addaction name="separator"/>
   <addaction name="action_Suchen"/>
   <addaction name="separator"/>
   <addaction name="actionMarkierterAngelplatz"/>
   <addaction name="actionAlleAngelpltze"/>
  </widget>
  <action name="actionBEenden">
   <property name="icon">
    <iconset resource="Angeltagebuch.qrc">
     <normaloff>:/beenden.png</normaloff>:/beenden.png</iconset>
   </property>
   <property name="text">
    <string>B&amp;eenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNeu">
   <property name="icon">
    <iconset resource="Angeltagebuch.qrc">
     <normaloff>:/neu.png</normaloff>:/neu.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Neu...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNdern">
   <property name="icon">
    <iconset resource="Angeltagebuch.qrc">
     <normaloff>:/aendern.png</normaloff>:/aendern.png</iconset>
   </property>
   <property name="text">
    <string>Ä&amp;ndern...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionLschen">
   <property name="icon">
    <iconset resource="Angeltagebuch.qrc">
     <normaloff>:/loeschen.png</normaloff>:/loeschen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Löschen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionMarkierterAngelplatz">
   <property name="icon">
    <iconset resource="Angeltagebuch.qrc">
     <normaloff>:/viewOne.png</normaloff>:/viewOne.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Markierter Angelplatz</string>
   </property>
  </action>
  <action name="actionAlleAngelpltze">
   <property name="icon">
    <iconset resource="Angeltagebuch.qrc">
     <normaloff>:/viewAll.png</normaloff>:/viewAll.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Alle Angelplätze</string>
   </property>
  </action>
  <action name="actionDeutsch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Deutsch</string>
   </property>
  </action>
  <action name="actionEnglisch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Englisch</string>
   </property>
  </action>
  <action name="actionWeiss">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Weiss</string>
   </property>
  </action>
  <action name="actionGelb">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Gelb</string>
   </property>
  </action>
  <action name="actionGraU">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gra&amp;u</string>
   </property>
  </action>
  <action name="actionGRn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grü&amp;n</string>
   </property>
  </action>
  <action name="actionRot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Rot</string>
   </property>
  </action>
  <action name="actionBlau">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Blau</string>
   </property>
  </action>
  <action name="actionSerBisch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ser&amp;bisch</string>
   </property>
  </action>
  <action name="action_Suchen">
   <property name="icon">
    <iconset resource="Angeltagebuch.qrc">
     <normaloff>:/suchen.png</normaloff>:/suchen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Suchen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Angeltagebuch.qrc"/>
 </resources>
 <connections/>
</ui>
